espnet2/bin/speechlm_inference.py

```diff
@@ -381,7 +381,8 @@ def inference(
                         logging.info(f"Save audio: {audio_path}")
                     
                     elif modality in ["text_bpe"]:
-                        writers[name].write(f"{example_name} {detokenized}")
+                        detokenized = detokenized.strip()
```

Comment:
@jctian98 Please let me know if this is fixed in your latest PR then I can remove this temporary modification.